src/mui-setup.js

```diff
@@ -12,11 +12,6 @@ export default function muiSetup(sproutBase) {
       },
     };
 
-    sproutBase.overrides.MuiTableBody.root = {
```

Comment:
could you fix it, please?

Reply:
You are setting the same thing the line above, or am I missing something?

Reply:
You are right.